From e7d4fda472bae31ac3c0992e48f4e3b96e6cb11d Mon Sep 17 00:00:00 2001
From: Gary Bisson <gary.bisson@boundarydevices.com>
Date: Tue, 23 Nov 2021 09:59:58 +0100
Subject: [PATCH] imx/start_isp.sh: customize for Boundary

- do not load the modules manually as they load automatically
- always use 1080p as the default resolution for Basler camera

Signed-off-by: Gary Bisson <gary.bisson@boundarydevices.com>
---
 imx/start_isp.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/imx/start_isp.sh b/imx/start_isp.sh
index 892b7c061..2911758b8 100755
--- a/imx/start_isp.sh
+++ b/imx/start_isp.sh
@@ -21,13 +21,13 @@ if [ $NR_DEVICE_TREE_BASLER -eq 1 ]; then
 
 		# Default configuration for Basler daA3840-30mc: basler_4k
 		# Available configurations: basler_4k, basler_1080p60, basler_4khdr, basler_1080p60hdr
-		exec ./run.sh -c basler_4k -lm
+		exec ./run.sh -c basler_1080p60
 
 	elif [ $NR_DEVICE_TREE_OV5640 -eq 1 ]; then
 
 		# Default configuration for Basler daA3840-30mc: basler_1080p60
 		# Available configurations: basler_1080p60, basler_1080p60hdr
-		exec ./run.sh -c basler_1080p60 -lm
+		exec ./run.sh -c basler_1080p60
 
 	fi
 
@@ -38,7 +38,7 @@ elif [ $NR_DEVICE_TREE_BASLER -eq 2 ]; then
 	cd $RUNTIME_DIR
 	# Default configuration for Basler daA3840-30mc: dual_basler_1080p60
 	# Available configurations: dual_basler_1080p60, dual_basler_1080p60hdr, dual_basler_1080p25hdr
-	exec ./run.sh -c dual_basler_1080p60 -lm
+	exec ./run.sh -c dual_basler_1080p60
 
 else
 	# no device tree found exit with code no device or address
-- 
2.33.0

